stdlib/builtins.pyi

```diff
@@ -1028,22 +1028,22 @@ else:
         flags: int = ...,
         dont_inherit: int = ...,
         optimize: int = ...,
-    ) -> Any: ...
+    ) -> object: ...
 
 def copyright() -> None: ...
 def credits() -> None: ...
-def delattr(__obj: Any, __name: str) -> None: ...
+def delattr(__obj: object, __name: str) -> None: ...
 def dir(__o: object = ...) -> list[str]: ...
 @overload
 def divmod(__x: SupportsDivMod[_T_contra, _T_co], __y: _T_contra) -> _T_co: ...
 @overload
 def divmod(__x: _T_contra, __y: SupportsRDivMod[_T_contra, _T_co]) -> _T_co: ...
 def eval(
-    __source: str | bytes | CodeType, __globals: dict[str, Any] | None = ..., __locals: Mapping[str, Any] | None = ...
+    __source: str | bytes | CodeType, __globals: dict[str, object] | None = ..., __locals: Mapping[str, object] | None = ...
```

Comment:
Changing the globals argument is problematic because of invariance. Mapping is covariant in its value type so it should be fine to change the locals argument, though.

Reply:
Huh, and I see it does in fact have to be a `dict` rather than any old mapping. What a pain.